src/qt/transactionrecord.h

```diff
@@ -20,12 +20,7 @@ struct WalletTxStatus;
 
 /** UI model for transaction status. The transaction status is the part of a transaction that will change over time.
  */
-class TransactionStatus
-{
-public:
-    TransactionStatus() : countsForBalance(false), sortKey(""),
-                          matures_in(0), status(Unconfirmed), depth(0), open_for(0)
-    { }
+struct TransactionStatus {
 
```

Comment:
b0ffe4e47c0500bcd54f82e6b88a0941ca nit if you retouch, can drop the extra newline

Reply:
Applying [`clang-format-diff.py`](https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy) after each commit is a good habit :)

Reply:
Done. Thanks.